### ISSUE TITLE
bugzillas are black

### ISSUE THREAD

**Issue by nekoswag** [QupZilla/qupzilla#2617]
System: openSUSE Tumbleweed 20180221, QupZilla 2.2.5

I'm running into a strange error lately, bugzilla pages are completely black when searching for something.
Example:
https://bugs.kde.org/buglist.cgi?no_redirect=1&quicksearch=dolphin
Shows a completely black page. I tried disabling AdBlock and pi-hole, to no effect. The same page renders just fine in Firefox.

**Comment by cranes-bill** [comment]
Did you try with clean profile?
Works fine here:
![capture du 2018-02-23 18 34 19](https://user-images.githubusercontent.com/5161278/36605126-75b1cf66-18c8-11e8-8cf6-dea921ef9c2b.png)

**Comment by nekoswag** [comment]
It works in a new profile. What could be the issue here?

**Comment by cranes-bill** [comment]
Something with the cache probably...
You can try to delete it in the old profile.

**Comment by nekoswag** [comment]
I set it to delete the cache on exit, but the problem persists.

**Comment by cranes-bill** [comment]
Also and HTML5 content?

**Comment by nekoswag** [comment]
Yes, cache, html5 content and cookies are all auto-deleted on exit.

**Comment by dufresnep** [comment]
Having same problem here on TrueOS (FreeBSD derivative).
Well, only all web content is black. Tab, address bar, menu is there and working.
Moving the cursor I can see sometimes a URL appears at the bottom.

Pressing the left button, and moving, some text appears and move with the mouse.

Application version 2.1.2
QtWebEngine version 5.7.1

Error I get in the terminal that started qupzilla:
[paul@trueos-6310] ~% qupzilla
lthemeengine: using lthemeengine plugin
[0223/190654:ERROR:network_change_notifier.cc(535)] Not implemented reached in static net::NetworkChangeNotifier *net::NetworkChangeNotifier::Create()
QupZilla: 0 extensions loaded
lthemeengine: D-Bus global menu: no
[0223/190655:ERROR:systeminfo.cc(79)] No function to get number of cores
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
AdBlockSearchTree: Inserting rule with filter len <= 0!
[0223/190804:ERROR:systeminfo.cc(79)] No function to get number of cores
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: ocsp.digicert.com
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: ocsp.digicert.com
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: crl4.digicert.com
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: ocsp.digicert.com
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: ocsp.digicert.com
[0223/190804:ERROR:nss_ocsp.cc(609)] No URLRequestContext for NSS HTTP handler. host: crl4.digicert.com
js: Unrecognized Content-Security-Policy directive 'worker-src'.

[0223/190809:ERROR:systeminfo.cc(79)] No function to get number of cores
js: Uncaught SecurityError: Blocked a frame with origin "https://tylerlh.github.io" from accessing a frame with origin "https://www.qupzilla.com". Protocols, domains, and ports must match.
js: Mixed Content: The page at 'https://tylerlh.github.io/github-latest-commits-widget/?username=QupZilla&repo=qupzilla&limit=5' was loaded over HTTPS, but requested an insecure stylesheet 'http://weloveiconfonts.com/api/?family=fontawesome'. This request has been blocked; the content must be served over HTTPS.
Falling back to using screens root_visual.
Falling back to using screens root_visual.
Falling back to using screens root_visual.
Falling back to using screens root_visual.
Falling back to using screens root_visual.
[0223/190901:ERROR:systeminfo.cc(79)] No function to get number of cores
js: Unrecognized Content-Security-Policy directive 'worker-src'.

QXcbConnection: XCB error: 3 (BadWindow), sequence: 9226, resource id: 12584203, major code: 40 (TranslateCoords), minor code: 0

**Comment by dufresnep** [comment]
Ok, I did read to fast.
This bug happens on some pages... mine happens on all pages.

**Comment by cranes-bill** [comment]
@dufresnep Your QupZilla version is no longer maintained as well as QtWebEngine version 5.7 is pretty buggy.
@nekoswag You can also try to backup the settings file to see if that caused by some setting.

**Comment by nekoswag** [comment]
I found the culprit. It happens when setting a Firefox User Agent.

`Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0`

setting this string will cause the above bugzilla exampel link to show a black page.
https://bugs.kde.org/buglist.cgi?no_redirect=1&quicksearch=dolphin
